I_have_no_vignette.py

```diff
@@ -0,0 +1,34 @@
+import random
+from datetime import date, timedelta, datetime
+import calendar
+
+
+def check_monday(date):
+    print(f'The random date is : {date.date()}')
+    day = calendar.day_name[date.weekday()]
+    if day == "Monday":
+        print("אין לי ויניגרט!")
+
+
+def random_date(date1, date2):         #The left date is bigger
```

Comment:
Also here, please add documantation and use better names for the function and
the variables.